statie/tests/Posts/Year2016/EventDispatcher/EventDispatchingTest.php

```diff
@@ -1,26 +0,0 @@
-<?php declare(strict_types=1);
-
-namespace Pehapkari\Statie\Tests\Posts\Year2016\EventDispatcher;
-
-use Pehapkari\Statie\Posts\Year2016\EventDispatcher\EventSubscriber\NotifyMeOnVideoPublishedEventSubscriber;
-use PHPUnit\Framework\TestCase;
-use Symfony\Component\EventDispatcher\EventDispatcher;
-
-final class EventDispatchingTest extends TestCase
-{
-    public function test(): void
-    {
-        $eventDispatcher = new EventDispatcher;
-        $notifyMeOnVideoPublishedEventSubscriber = new NotifyMeOnVideoPublishedEventSubscriber;
-        $eventDispatcher->addSubscriber($notifyMeOnVideoPublishedEventSubscriber);
-
-        // nothing happened, default value
-        $this->assertFalse($notifyMeOnVideoPublishedEventSubscriber->isUserNotified());
-
-        // this calls our Subscriber
-        $eventDispatcher->dispatch('youtube.newVideoPublished');
-
-        // now it's changed
-        $this->assertTrue($notifyMeOnVideoPublishedEventSubscriber->isUserNotified());
-    }
-}
```

Comment:
The "string" approach is deprecated since Symfony 4.3 - https://symfony.com/blog/new-in-symfony-4-3-simpler-event-dispatching